Dockerfile

```diff
@@ -1,22 +1,15 @@
 # TODO: Upgrade to a more recent fedora version
 FROM fedora:32
 
-# The caller should build a Freshmaker RPM package and then pass it in this arg.
-ARG cacert_url=undefined
-ARG commitid=undefined
-
 LABEL \
     name="Freshmaker application" \
     vendor="Freshmaker developers" \
     license="GPLv2+" \
     build-date="" \
```

Comment:
The trailing "\" is unnecessary now. we can remove the `build-date` line as well since it's just an empty string.

Reply:
Ok, I will do it